src/Stepic/PrintTextPerRole.java

```diff
@@ -11,6 +12,19 @@ public static void main(String[] args) {
                 "Аммос Федорович: Вот те на!",
                 "Артемий Филиппович: Вот не было заботы, так подай!",
                 "Лука Лукич: Господи боже! еще и с секретным предписаньем!"};
-        
+        String[] forSearching=new String[roles.length];
+        for(int i =0;i< roles.length;i++){
```

Comment:
Could you move it in one cycle?

Reply:
without using the additional variable forSearching?
or shove everything into the second cycle?